### PR TITLE
Update build scripts for 1.9.20

### PR DESCRIPTION
**Pull request by abdulowork**
This PR updates the build scripts for the 1.9.20 release. The tutorial is updated [here](https://github.com/JetBrains/kotlin-web-site/pull/3824)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -8,7 +8,4 @@ kotlin.code.style=official
 kotlin.mpp.androidSourceSetLayoutVersion=2
 
 #Android
-android.useAndroidX=true
-
-#Sql Delight Version
-sqlDelightVersion=1.5.5
```
Comment (abdulowork) [inline gradle.properties:14]: https://github.com/JetBrains/kotlin-web-site/pull/3824#discussion_r1355103499

**Review by pahill — APPROVED** [review]
LGTM, let's sync the merge with the Technical Writing team updating the web docs please

**Review by danil-pavlov — APPROVED** [review]
Since these changes concern 1.9.20, let's merge them on the day of the 1.9.20 release

**Comment by pahill** [comment]
Thanks, we've updated the tutorial after some big refactorings. Thank you for taking the time to file this correction,

**Pull request closed without merging**